documentation/general/torn-sdk.md

```diff
@@ -35,7 +46,20 @@ Specifically we will be
 1. Changing msbuild to use a compiler from the .NET SDK
 2. Changing Visual Studio to use analyzers from Visual Studio
 
-In addition to making our builds more reliable this will also massively simplify our analyzer development strategy. Analyzers following this model can always target the latest Roslyn version without the need for complicated multi-targeting.
+In addition to making our builds more reliable this will also massively simplify our [analyzer Development strategy][sdk-lifecycle]. Analyzers following this model can always target the latest Roslyn version without the need for complicated multi-targeting.
+
+## Motivations
+
+There are a number of scenarios where customers end up in a torn state. The most common is when customers have a CI setup with the following items:
+
+1. They use a task like [actions/setup-dotnet][setup-dotnet] to install the .NET SDK and use a flexible version format like `8.x`.
+2. They use `msbuild` to actually drive their build. This `msbuild` comes from the Visual Studio installed on the machine image.
+
+This means that CI systems are updated to the latest .NET SDK virtually as soon as we release them. However the version of Visual Studio is updated much later as CI images usually take several weeks to upgrade to a new Visual Studio version. This is a very common CI setup and means a significant number of our customers end up in a torn state for several weeks.
+
+Another reason is that teams use older Visual Studio versions due to internal constraints: like an organizational policy. At the same time they install the latest .NET SDK which puts them into a torn state.
+
+This also hits any customer that uses a preview version of .NET SDK. These inherently represent a torn SDK state because they almost never match the compiler in Visual Studio This results in blockers for big teams like Bing from testing out our previews.
```

Comment:
```suggestion
This also hits any customer that uses a preview version of .NET SDK. These inherently represent a torn SDK state because they almost never match the compiler in Visual Studio. This results in blockers for big teams like Bing from testing out our previews.
```